### DISCUSSION TITLE
Why is the app called "Trilium"?

### DISCUSSION THREAD

**Discussion by sathyamvellal** [TriliumNext/Trilium#4641] [General]
I'm not sure if this was asked before or is answered in the wiki (couldn't find it in a quick glance). I wanted to understand why the app was called "Trilium". Is anyone in the community aware of it?

**Comment by sigaloid** [comment]
Here's the origin: https://github.com/zadam/trilium/issues/1180#issuecomment-681852967

**Reply by meichthys** [reply]
Also added to the wiki for reference :)
https://github.com/zadam/trilium/wiki
